gdal_subsetter/transform.py

```diff
@@ -696,7 +696,10 @@ def recolor(self, layerid, srcfile, dstdir):
                             if relatedUrl.type == 'Color Map':
                                 colormap = '/vsicurl/' + relatedUrl.url
                                 discrete = True
-        if colormap is None and ('png' in fmt.mime or 'jpeg' in fmt.mime):
+
+        # Don't color tiffs
+        colorable_mime_types = ['image/png', 'image/jpeg']
```

Comment:
Even though this variable is only being used once, I like it because it makes it really clear what the next line is checking for. (Also it's easy to update the list without having to make the line below jump through formatting hoops for line length)